tests/test_sqlalchemy.py

```diff
@@ -0,0 +1,62 @@
+from coaster.sqlalchemy import BaseMixin
+from coaster.db import db
+from sqlalchemy import Column, Unicode
+from baseframe import _, __
+from baseframe import baseframe
+import baseframe.forms as forms
+import baseframe.forms.sqlalchemy as forms_sqlachemy
+from .fixtures import BaseframeTestCase, app1, app2
+
+
+class Container(BaseMixin, db.Model):
+    __tablename__ = 'container'
+    name = Column(Unicode(80), nullable=True)
+    title = Column(Unicode(80), nullable=True)
+    content = Column(Unicode(250))
+
+
+class ContainerForm(forms.Form):
+    name = forms.StringField(__("Name"), validators=[forms_sqlachemy.AvailableName()])
+    title = forms.StringField(__("Title"), validators=[forms_sqlachemy.AvailableAttr('title')])
+    content = forms.TextAreaField(__("Content"))
+
+
+class FormSQLAlchemyTestCase(BaseframeTestCase):
+    def setUp(self):
```

Comment:
Don't inherit from `BaseframeTestCase` as that messes with `self.app`. Just add the `init_app` call here.